main.md

```diff
@@ -88,7 +91,7 @@ BASE64URL denotes the URL-safe Base64 encoding without padding as defined in
 
 # Terminology
 
- * A **SD-JWT** is a signed JWT [@!RFC7515], i.e., a JWS, that contains claims
+ * A **SD-JWT** is a signed JWT [@!RFC7519], i.e., a JWS [@!RFC7515], that contains claims
```

Comment:
SD-JWT itself does not have to be signed. That’s why we changed SD-JWS to SD-JWT, right?

Reply:
As mentioned in another PR:
Should we add a generic SD-JSON? This way the approach can be also used for JSON-LD or any other JSON object (signed or unsigned).